Dockerfile

```diff
@@ -31,9 +31,6 @@ WORKDIR /app
 RUN apk add proxychains-ng
 
 ENV PROXY_URL=""
-ENV OPENAI_API_KEY=""
```

Comment:
It would be better commenting the env variables so that people would know what env to set when using this docker images

Reply:
maybe those documentations can stay in the readme file?